### PR TITLE
Add platoniq footer

### PR DESCRIPTION
**Pull request by verarojman**

### CODE REVIEW

**Review by microstudi — APPROVED** [review]

File: app/views/layouts/decidim/_mini_footer.html.erb [hunk 1]
```diff
@@ -1,5 +1 @@
-<% if fearlesscities_site? %>
-  <%= render partial: "layouts/fearlesscities/minifooter" %>
-<% else %>
-  <%= render partial: "layouts/decidim/minifooter" %>
-<% end %>
+<%= render partial: "platoniq/mini_footer" %>
```
Comment (davidbeig) [inline app/views/layouts/decidim/_mini_footer.html.erb:1]: If we remove the partials for the different tennants, how can we add the images of the different organizations to the footer?

Comment (verarojman) [reply app/views/layouts/decidim/_mini_footer.html.erb:1]: 🤔 Is it the case here? I thought there were no custom images in either of the footers

Comment (verarojman) [reply app/views/layouts/decidim/_mini_footer.html.erb:1]: We can change it later if necessary

Comment (davidbeig) [reply app/views/layouts/decidim/_mini_footer.html.erb:1]: No, but I thought that maybe we can add the logo of bcnencomu. They might ask for it, but ofc we can change it later!

**Review by davidbeig — APPROVED** [review]

**Pull request merged**